AcadeMeData/tests.py

```diff
@@ -1,3 +1,9 @@
-# from django.test import TestCase
+import pytest
+from AcadeMeData.models import University
 
-# Create your tests here.
+
+class TestUniversityModel(pytest):
+    @pytest
+    def test_get_university_by_id(self):
```

Comment:
Do you think this method makes sense ? 
I would consider searching university by name, for a start